src/server/errors.ts

```diff
@@ -10,6 +10,13 @@ export class InternalServerError extends BaseError {
     }
 }
 
+export class CrudTypeNotFound<T> extends BaseError {
+    constructor() {
+        let ctor: { new (): T };
```

Comment:
@Evilart86 ctor -> constructor ? pls, don't use abbreviations in names. Also use **const** if variable not change

Reply:
Ok, i'll rename it